plutus-tx-plugin/test/StdLib/Spec.hs

```diff
@@ -115,8 +115,12 @@ testReduce = Hedgehog.property $ do
 testOrd :: Property
 testOrd = Hedgehog.property $ do
     let gen = Gen.integral (Range.linear (-10000) 100000)
-    n1 <- Hedgehog.forAll $ (%) <$> gen <*> gen
-    n2 <- Hedgehog.forAll $ (%) <$> gen <*> gen
+        -- Ratio must have non-zero denominator or else an ArithException will be thrown.
+        gen' =
+          Gen.choice
+            [Gen.integral (Range.linear (-10000) (-1)), Gen.integral (Range.linear 1 100000)]
```

Comment:
[`filter`](https://hackage.haskell.org/package/hedgehog-1.4/docs/Hedgehog-Gen.html#v:filter) would perhaps be a bit more readable, but this is fine too.

Reply:
Thanks, `filter` is much better.